test/medium_test_exhaust.py

```diff
@@ -31,8 +33,7 @@ def make_models(self, tmpdir, make_rmf=False):
         mod_dir = os.path.join(tmpdir, 'modeling')
         shutil.copytree(self.get_input_file_name('modeling'), mod_dir)
 
-        self.run_python_module(
-            select_good,
+        self.run_python_module(select_good,
```

Comment:
Not sure why you changed this. The original was correctly formatted; the new code is not. (Same deal in lots of other places below; I'm not going to comment on each one.)